pkg/syncer/metadatasyncer.go

```diff
@@ -41,6 +44,32 @@ func NewInformer() *MetadataSyncInformer {
 	return &MetadataSyncInformer{}
 }
 
+// getFullSyncIntervalInMin return the FullSyncInterval
+// If enviroment variable FULL_SYNC_INTERVAL_MINUTES is set and valid,
+// return the interval value read from enviroment variable
+// otherwise, use the default value 30 minutes
+func getFullSyncIntervalInMin() int {
+	fullSyncIntervalInMin := defaultFullSyncIntervalInMin
+	if v := os.Getenv("FULL_SYNC_INTERVAL_MINUTES"); v != "" {
```

Comment:
@divyenpatel What are your thoughts on this style? I had suggested to make `FULL_SYNC_INTERVAL_MINUTES` a `const`, but instead it's been embedded into a const log string prefix in `pkg/syncer/types.go`. It works that way, just not a pattern I usually see. Your call...

Once you are happy with this, and the two commits are squashed, this looks ready to me.

Reply:
@codenrhoden Do you mean define something like 
```
const envFullSyncIntervalMinutes = "FULL_SYNC_INTERVAL_MINUTES", 
```
and then
use
```
 if v := os.Getenv(envFullSyncIntervalMinutes); v != "" { ....
```

Reply:
@lipingxue yes const would be good. also lets correct fullSyncIntervalLogMessagePrefix as commented at https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/61/files#r323369929

Reply:
Done.